package.nls.json

```diff
@@ -1,5 +1,6 @@
 {
     "python-envs.defaultEnvManager.description": "The default environment manager for creating and managing environments.",
+    "python-envs.injectEnvVarsInTerminals.description": "This setting defaults to false and toggles if environment variables found in the workspace are set for new terminals.",
```

Comment:
@cwebster-99 interested your thoughts here on the name of the setting and the description. Would appreciate your rewrite since this is not very good

Reply:
hold on this- rewriting implementation